hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/storage/DiskBalancerConfiguration.java

```diff
@@ -146,15 +146,31 @@ public Class<?> getContainerChoosingPolicyClass() {
     return containerChoosingPolicyClass;
   }
 
+  @Config(key = "stop.after.disk.even",
+      type = ConfigType.BOOLEAN,
+      defaultValue = "true",
+      tags = {ConfigTag.DISKBALANCER},
+      description = "If true, the DiskBalancer will automatically stop once disks are balanced.")
+  private boolean stopAfterDiskEven = true;
+
+  public boolean getStopAfterDiskEven() {
```

Comment:
Please rename this to `isStopAfterDiskEven()`.
```suggestion
  public boolean isStopAfterDiskEven() {
```